packages/legend-art/style/base/_panel.scss

```diff
@@ -179,6 +179,59 @@
   }
 }
 
+.panel__container {
+  height: 100%;
+  width: 100%;
+
+  &--border-top {
+    border-top: 0.1rem solid var(--color-dark-grey-100);
+  }
+
+  &--border-right {
+    border-right: 0.1rem solid var(--color-dark-grey-100);
+  }
+}
+
+.panel__explorer {
+  &__panel {
```

Comment:
this one doesn't seem to be in use, could you scan these and make sure for me you don't have anything extraneous?

Reply:
yes! ^^;